features/steps/command/shell.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Running shell commands
+
+  You can run an *ad hoc* script with the following step
+  - `When I run the following script:`
+
+  Or you can run shell commands with:
+  - `I run the following (bash|zsh|fish|dash)? commands`
+
+  Background:
+    Given I use a fixture named "cli-app"
+
+  Scenario: Running script
+    Given a file named "features/shell.feature" with:
+    """
+    Feature: Running scripts
+      Scenario: Running ruby script
+        When I run the following commands:
+        \"\"\"bash
+        #!/usr/bin/env ruby
+
+        puts "Hello"
+        \"\"\"
+        Then the output should contain exactly "Hello"
+
+      Scenario: Running python script
+        When I run the following commands:
+        \"\"\"bash
+        #!/usr/bin/env python
+
```

Comment:
Splitting up those tests on the "highest" level makes it easier to troubleshooting our test suite.
~~~ cucumber
Scenario: Running ruby script
  Given a file named "features/shell.feature" with:
  """
  Feature: Running script
    Scenario: Running script
      When I run the following commands:
      \"\"\"bash
      #!/usr/bin/env ruby

      puts "Hello"
      \"\"\"
      Then the output should contain exactly "Hello"
~~~

~~~ cucumber
Scenario: Running python script
  Given a file named "features/shell.feature" with:
  """
  Feature: Running script
    Scenario: Running script
      When I run the following commands:
      \"\"\"bash
      #!/usr/bin/env python

      print("Hello")
      \"\"\"
      Then the output should contain exactly "Hello"
~~~